src/components/Settings.vue

```diff
@@ -120,6 +120,20 @@
           <input type="text" :name="'providers['+name+'][guilds]'" v-model="provider.guilds"/>
         </label>
       </template>
+      <template v-if="provider.hasGroupMapping">
+        <button class="group-mapping-add" type="button" @click="provider.groupMapping.push({foreign: '', local: ''})">
```

Comment:
For some reason the existence of the Add Group Mapping button widens the Discord box and breaks the layout. I do not understand. I've already spent 40 minutes looking at it.

Seriously what is this BS

![srs](https://github.com/zorn-v/nextcloud-social-login/assets/987487/7c4227b3-18e0-487c-8242-e27d55c1b0cc)

Fix the CSS here.